embedded-bins/kine/Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN cd / && git -c advice.detachedHead=false clone -b v$VERSION --depth=1 https:
 WORKDIR /kine
 RUN go version
 RUN CGO_ENABLED=${BUILD_GO_CGO_ENABLED} \
-    go build \
+    CGO_CFLAGS="-DSQLITE_ENABLE_DBSTAT_VTAB=1 -DSQLITE_USE_ALLOCA=1" go build \
```

Comment:
Probably makes sense to add those flags to `Makefile.variables`, just like it's done for the other build flags, à la `BUILD_GO_CGO_CFLAGS`. Would you mind to give it a try?

Moreover a comment with a link to the upstream build flags might be handy: https://github.com/k3s-io/kine/blob/v0.9.1/scripts/build#L16

Reply:
done. I also added the variables for the other components but commented to make it uniform.